tests/features/site.feature

```diff
@@ -2,7 +2,8 @@ Feature: site
 
   Scenario: Site Clone Environment
     @vcr site-clone-env
-    Given a site named "[[test_site_name]]"
+    Given I am authenticated
+    And a site named "[[test_site_name]]"
```

Comment:
This will never pass. aSiteNamed never replaces the placeholder [[test_site_name]]